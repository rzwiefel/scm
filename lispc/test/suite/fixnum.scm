(write (number? 0))
(write 0)
(write 42)
