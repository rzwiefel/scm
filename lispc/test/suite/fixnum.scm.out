#t
0
42
